test/write.jl

```diff
@@ -34,10 +34,12 @@
         "EXIF_TAG_SATURATION"=>"Normal",
         "EXIF_TAG_CONTRAST"=>"Normal",
         "EXIF_TAG_SHARPNESS"=>"Normal",
-        "EXIF_TAG_COMPONENTS_CONFIGURATION"=> "Y Cb Cr -"
+        "EXIF_TAG_COMPONENTS_CONFIGURATION"=> "Y Cb Cr -",
+        "EXIF_TAG_USER_COMMENT"=>"Dummy comment",
+        # "EXIF_TAG_MAKER_NOTE"=>"Maker Note",  # TODO: support `EXIF_TAG_MAKER_NOTE`
     )
     path = joinpath(tempdir(), "tmp.jpg")
-    write_tags(path; img, tags=tags)
+    write_tags(path; img, tags)
```

Comment:
Any opinion on tests?  how could we make them more robust?

Reply:
I think there currently are ok, tests are always incremental :)